plugin/plugin.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2020 The Knative Authors
```

Comment:
Referred to latest [Knative/Serving repo](https://github.com/knative/serving/blob/3b6b40aec07654c1d2d411397122ae279a4ccfe4/pkg/reconciler/gc/reconciler.go#L2), the Copyright is 
```
Copyright 2020 The Knative Authors
```
Please help to fix it, and I've updated others in https://github.com/knative-sandbox/kn-plugin-admin/pull/41

Reply:
Done in https://github.com/knative-sandbox/kn-plugin-admin/pull/38/commits/4b20ec46ae2bd73190a611948f0a358f087941ea. Sorry that it took me so long to fix it. :)